neptune/experiments.py

```diff
@@ -129,36 +130,43 @@ def system_properties(self):
 
             Get experiment system properties.
 
-            >>> experiment.system_properties
+            >>> experiment.get_system_properties
 
         Note:
             The list of supported system properties may change over time.
 
         """
-        return self._simple_dict_to_dataframe(self._leaderboard_entry.system_properties)
-
-    @property
-    def tags(self):
-        return self._leaderboard_entry.tags
+        experiment = self._client.get_experiment(self._internal_id)
+        return self._simple_dict_to_dataframe({
+            'id': experiment.shortId,
+            'name': experiment.name,
+            'created': experiment.timeOfCreation,
+            'finished': experiment.timeOfCompletion,
+            'running_time': experiment.runningTime,
+            'owner': experiment.owner,
+            'size': experiment.storageSize,
+            'tags': experiment.tags,
+            'notes': experiment.description
+        })
+
+    def get_tags(self):
+        return self._client.get_experiment(self._internal_id).tags
 
     def append_tag(self, tag):
```

Comment:
It would be nice if we could pass a list of tags here.

Reply:
It's an interface defined by Jakub. He said that it's more convenient for a typical user to add and remove tags one by one.